src/config.rs

```diff
@@ -31,7 +31,9 @@ impl Display for OverrideReason {
     }
 }
 
+#[derive(Clone)]
```

Comment:
I'm not 100% on if it's completely safe to do this, given `settings_file` has interior mutability.

As such this entire change to `Cfg` is a bit squiffy.